packages/network-controller/src/index.ts

```diff
@@ -1,5 +1,42 @@
 export type { AutoManagedNetworkClient } from './create-auto-managed-network-client';
-export * from './NetworkController';
+export type {
+  Block,
+  NetworkMetadata,
+  NetworkConfiguration,
+  BuiltInNetworkClientId,
+  CustomNetworkClientId,
+  NetworkClientId,
+  NetworksMetadata,
+  NetworkState,
+  BlockTrackerProxy,
+  ProviderProxy,
+  AddNetworkFields,
+  UpdateNetworkFields,
```

Comment:
```suggestion
  UpdateNetworkFields,
  RpcEndpoint,
```

`RpcEndpoint` would be convenient to export